codegen/collector/executor.go

```diff
@@ -108,11 +108,12 @@ func (o *OpenApiProtocExecutor) Execute(protoFile string, toFile string, imports
 	_, fileName := filepath.Split(protoFile)
 	directoryName := fileName[0 : len(fileName)-len(filepath.Ext(fileName))]
 
-	baseArgument := "--openapi_out=yaml=true,single_file=false,include_description=true,multiline_description=true,proto_oneof=true,int_native=true"
+	baseArgument := "--openapi_out=yaml=true,single_file=false,multiline_description=true,proto_oneof=true,int_native=true"
```

Comment:
is there a test suite for this?
or can you link to a PR that shows the GME CRDs diff after this change?

Reply:
https://github.com/solo-io/gloo-mesh-enterprise/pull/18641